NOTICE

```diff
@@ -6,3 +6,7 @@ Xayn AG (https://www.xayn.com/).
 
 Except as otherwise noted (below and/or in individual files), this project is
 licensed under the GNU Affero General Publc License v3.0 only <LICENSE>.
+
+- discovery_engine_core/ai/rubert-tokenizer
+  This crate is a fork of https://github.com/huggingface/tokenizers and is licensed
+  under the Apache License, Version 2.0 <discovery_engine_core/ai/rubert-tokenizer/LICENSE>.
```

Comment:
Added this because of the different license of rubert-tokenizer

Reply:
do we already know how to handle the ci license check? are we going to exclude this directory from the check or are we going to check for various license headers?